File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Facade.Common/Interfaces/IProtocolImporter.cs
using System.IO;
using System.Xml;
using VecompSoftware.DocSuiteWeb.Data;

namespace VecompSoftware.DocSuiteWeb.Facade.Common.Interfaces
{
    public interface IProtocolImporter
    {
        Protocol Import(XmlDocument xml, FileInfo pdf, ProtocolTemplate template);
        bool ClassificationManaged { get; }
    }
}

File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/Code/IProtocolImporter.vb
Imports System.IO
Imports VecompSoftware.DocSuiteWeb.Data
Imports System.Xml

Public Interface IProtocolImporter

    Function Import(ByVal xml As XmlDocument, ByVal pdf As FileInfo, ByVal template As ProtocolTemplate) As Protocol

    ReadOnly Property ClassificationManaged() As Boolean

End Interface